ghmirror/core/mirror_requests.py

```diff
@@ -78,10 +136,7 @@ def online_request(method, url, auth, data=None, url_params=None):
     headers = {}
     parameters = url_params.to_dict() if url_params is not None else {}
 
-    per_page_elements = _get_elements_per_page(url_params)
-
-    if per_page_elements is None:
-        per_page_elements = PER_PAGE_ELEMENTS
+    if _get_elements_per_page(url_params) is None:
```

Comment:
how is this related to the overall change?

Reply:
I needed the refactor of this as `online_request` function was using the maximum number of local variables that Pylint allow, and to add the `valid_request` I did this